snippets/card-product.liquid

```diff
@@ -118,7 +118,33 @@
               </a>
             </h3>
           </div>
-          <div class="card__badge {{ settings.badge_position }}">
+
+          {% assign CustomTags = settings.custom_badges  | newline_to_br | split: '<br />' %}
+          {% assign tagLabel = 0 %}
+          {% assign hasTag = 0 %}
+
+          {% for tag in CustomTags %}
+            {% assign tagSplit = tag | split: ':' %}
+            {% assign pTag = tagSplit.first %}
+            {% assign pLabel = tagSplit.last %}
+
+            {% assign hasTag = card_product.tags | where: pTag %}
+
+            {% if hasTag.size > 0 %}
+              {% assign tagLabel = pLabel %}
+            {% endif %}
+          {% endfor %}
```

Comment:
I don't think all of this is necessary. We don't need this complex customTag structure. The "partner pricing" badge shows up if product.type == 'Wholesale'.